private/numerics/functions/elliptic.rkt

```diff
@@ -0,0 +1,372 @@
+#lang racket/base
+
+;; Consult "Numerical Computation of Real or Complex Elliptic
+;; Integrals" Carlson (1994) http://arxiv.org/pdf/math/9409227v1.pdf
+;;
+;; The algorithm in scmutils seems similar to what is in that paper,
+;; but slightly different.
+;;
+;; Consider also exporting under the more familiar names Rf, Rj, Rc,
+;; Rd. Is there any input method that gets those names as subscripts.
```

Comment:
Let's provide `R_f`, etc. (I think we should eventually do this for all subscripted identifiers -- offer ascii alternatives.)